### PR TITLE
Changing JIT perf test names to fix how they are organized on BenchView

### PR DESCRIPTION
**Pull request by jorive**
The names used on the BenchView reporting site are taken from the fully qualified names of the methods running the benchmarks (marked with the xunit performance attribute [Benchmark]). Before these changes, a lot of the classes/methods were in the global namespace, and all the tests would show up in a long list. Now, after these changes, tests are grouped.

In addition, I append the all the BenchView results into a single file and upload it once.

### CODE REVIEW

File: tests/src/JIT/Performance/CodeQuality/V8/DeltaBlue/DeltaBlue.cs [hunk 1]
```diff
@@ -24,13 +24,15 @@
...
 
-/* 
+/*
 Strengths are used to measure the relative importance of constraints.
 New strengths may be inserted in the strength hierarchy without
 disrupting current constraints.  Strengths cannot be created outside
 this class, so pointer comparison can be used for value comparison.
 */
 
+namespace V8
```
Comment (dpodder) [inline tests/src/JIT/Performance/CodeQuality/V8/DeltaBlue/DeltaBlue.cs]: Is this left as `V8` instead of `V8.DeltaBlue` on purpose?

Comment (dpodder) [inline tests/src/JIT/Performance/CodeQuality/V8/DeltaBlue/DeltaBlue.cs]: @jorive That's fine then to leave as-is; thanks for clarifying!

**Comment by dpodder** [comment]
@jorive Thanks for the changes! Looks good now. Only issue left is to fix the test issue, I think. @DrewScoggins PTAL as well.

**Comment by jorive** [comment]
@dotnet-bot test Windows_NT perf

**Comment by dpodder** [comment]
@dotnet-bot test Windows_NT perf

**Comment by jorive** [comment]
@dotnet-bot test Windows_NT CoreCLR Perf Tests

**Comment by brianrob** [comment]
@dotnet-bot test Windows_NT CoreCLR Perf Tests

**Comment by mmitche** [comment]
@dotnet-bot help

**Comment by mmitche** [comment]
@dotnet-bot test Windows_NT perf

**Comment by jorive** [comment]
@mmitche, @dpodder, @DrewScoggins I updated the perf.groovy to make sure we are running with the latest BenchView tools.

**Comment by DrewScoggins** [comment]
In the latest tools did you make a change to make sure that the file exists before we append?  It looks like that is why the PR runs were failing.

**Comment by jorive** [comment]
@DrewScoggins yes, I updated the tools (fixed the append option), but the nuget command did not pull the latest scripts. It was caused by the -ExcludeVersion option (if folder exist, then skip). It was not downloading the latest version.

This time, I moved where we download to, and verify that if the folder exist, then remove it and download it again.

**Comment by DrewScoggins** [comment]
LGTM

**Comment by dpodder** [comment]
@dotnet-bot test Windows_NT perf

**Comment by mmitche** [comment]
@dotnet-bot test Windows_NT perf

**Comment by jorive** [comment]
@dotnet-bot test Windows_NT perf

**Comment by jorive** [comment]
@dotnet-bot test Windows_NT perf

**Comment by DrewScoggins** [comment]
All the tests have passed, this LGTM.

**Pull request merged**